src/components/WhoWeAre/WhoWeAre.module.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../assets/styles/theme";
+
+.container {
+    line-height: 1.6;
+    width: 100%;
+    margin-top: 30px;
+    overflow-x: hidden;
+}
+// LIGHT THEME
+
+.h1Light {
+    font: 20px $helvetica;
+    font-weight: bold;
+    padding-top: 26px;
+    width: 100%;
+    margin-bottom: 10px;
+    margin-left: 18px;
+}
+.pLight {
+    font: 16px $helvetica;
+    width: 90%;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-left: 18px;
+}
+.backgroundLight {
+    width: 100%;
+    margin: 0 auto;
+    background-clip: content-box;
+    border-top-right-radius: 150px 48px;
+    background-color: rgba(0, 0, 0, 0.49) ;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.25);
+}
+
+//DARK THEME
+
+.h1Dark {
```

Comment:
to z https://discord.com/channels/894595800343134248/1183889743621136404/1196812184378949664